### DISCUSSION TITLE
如何在plugin获取 所有组件 所有demos？

### DISCUSSION THREAD

**Discussion by mwy516** [umijs/dumi#2054] [Q&A]
如何在plugin获取 所有组件 所有demos？
例如
dumi1中的
import ComponentConfig from '@@/dumi/config'
import DemosConfig from '@@/dumi/demos'

**Comment by PeachScript** [comment]
ref: https://d.umijs.org/theme/api#usesitedata

但 v2.3.0 正式版马上要改 demo 数据获取的方式了，建议直接用 next tag 指向的版本，使用 `getFullDemos` 拿
